### ISSUE TITLE
Improve documentation on Literals/constants and class attributes

### ISSUE THREAD

**Issue by Josverl** [Josverl/micropython-stubber#209]
Currently the docstubs documentation wrt to constant and class attributes is written to the stub file as comments.
while this is documented as a supported format ( somewhere) it does not seem to work in practice.

**Describe the solution you'd like**
replace the prefixed comments with 'attribute docstrings' 

```
# Defined within some constants/best_numbers.py
THE_BEST_NUMBER: int = 42
"""The best number known. 

This extensive documentation explains why this particular
value is the best. It all started when....
"""
``` 
See: https://github.com/microsoft/pylance-release/issues/1576

**Comment by Josverl** [comment]
src\stubber\rst\output_dict.py
```
            _docstr = docstr
            if autoindent:
                line = spaces(self._indent + self._body) + line
                if len(_docstr):
                    # TODO:
                    #  - add docstring after line defining constant
                    #  - if len = 1 add single " quotes before & after
                    #  - if len > 1 add triple quotes before & after
                    # add docstr as comment before constant
                    _docstr = [spaces(self._indent + self._body) + "# " + l for l in _docstr]
            if len(_docstr):
                self.update({"constants": self["constants"] + _docstr + [line]})
            else:
                self.update({"constants": self["constants"] + [line]})

```

**Comment by Josverl** [comment]
Resolved by Commit: 4da1202a277296e989bc07a097f3a3c08392edbb
release 1.7.1
